physics/gravitational_force.cpp

```diff
@@ -0,0 +1,36 @@
+/**
+ * @file gravitational_force.cpp
```

Comment:
No need to add the filename. It's best to let Doxygen automatically detect it, otherwise, you'd have to specify the whole directory as well.
```suggestion
 * @file
```